src/ctapipe/irf/select.py

```diff
@@ -0,0 +1,298 @@
+                selected = self.make_derived_columns(selected, obs_conf)
+                bits.append(selected)
+                n_raw_events += len(events)
+
+            bits.append(header)  # Putting it last ensures the correct metadata is used
+            table = vstack(bits, join_type="exact", metadata_conflicts="silent")
+            return table, n_raw_events, meta
+
+    def get_metadata(
+        self, loader: TableLoader, obs_time: u.Quantity
+    ) -> tuple[SimulatedEventsInfo, PowerLaw, Table]:
+        obs = loader.read_observation_information()
+        sim = loader.read_simulation_configuration()
+        show = loader.read_shower_distribution()
+
+        for itm in ["spectral_index", "energy_range_min", "energy_range_max"]:
+            if len(np.unique(sim[itm])) > 1:
+                raise NotImplementedError(
+                    f"Unsupported: '{itm}' differs across simulation runs"
+                )
+
+        sim_info = SimulatedEventsInfo(
+            n_showers=show["n_entries"].sum(),
+            energy_min=sim["energy_range_min"].quantity[0],
+            energy_max=sim["energy_range_max"].quantity[0],
+            max_impact=sim["max_scatter_range"].quantity[0],
+            spectral_index=sim["spectral_index"][0],
+            viewcone_max=sim["max_viewcone_radius"].quantity[0],
+            viewcone_min=sim["min_viewcone_radius"].quantity[0],
+        )
+
+        return (
+            sim_info,
+            PowerLaw.from_simulation(sim_info, obstime=obs_time),
+            obs,
+        )
+
+    def make_derived_columns(self, events: QTable, obs_conf: Table) -> QTable:
+        if obs_conf["subarray_pointing_lat"].std() < 1e-3:
+            assert all(obs_conf["subarray_pointing_frame"] == 0)
+            # Lets suppose 0 means ALTAZ
+            events["pointing_alt"] = obs_conf["subarray_pointing_lat"][0] * u.deg
+            events["pointing_az"] = obs_conf["subarray_pointing_lon"][0] * u.deg
+        else:
+            raise NotImplementedError(
+                "No support for making irfs from varying pointings yet"
+            )
+
+        events["theta"] = calculate_theta(
+            events,
+            assumed_source_az=events["true_az"],
+            assumed_source_alt=events["true_alt"],
+        )
+        events["true_source_fov_offset"] = calculate_source_fov_offset(
+            events, prefix="true"
+        )
+        events["reco_source_fov_offset"] = calculate_source_fov_offset(
+            events, prefix="reco"
+        )
+
+        altaz = AltAz()
+        pointing = SkyCoord(
+            alt=events["pointing_alt"], az=events["pointing_az"], frame=altaz
+        )
+        reco = SkyCoord(
+            alt=events["reco_alt"],
+            az=events["reco_az"],
+            frame=altaz,
+        )
+        nominal = NominalFrame(origin=pointing)
+        reco_nominal = reco.transform_to(nominal)
+        events["reco_fov_lon"] = -reco_nominal.fov_lon  # minus for GADF
```

Comment:
I think we should add a new frame in `ctapipe.coordinates` called `GADFFovCoordinatesAltAz` or similar instead of doing this here by hand.

Reply:
Or just make the `NominalFrame` compatible with `GADF`, see #1843

Reply:
I would be in favor of making the existing frames compatible with GADF, but probably in a separate PR.